tests/test_entities.py

```diff
@@ -325,7 +325,7 @@ def test_id_and_which(self):
             (entities.Host, 'packages'),
             (entities.Host, 'puppetclass_ids'),
             (entities.Host, 'smart_class_parameters'),
-            (entities.Host, 'list_ansible_roles'),
+            (entities.Host, 'ansible_roles'),
```

Comment:
@sambible can you please try running this test case locally and see if that works for you, I purposely changed that to `list_ansible_roles` as I think we are putting the methods here and it worked for me.

Reply:
Yea, I've run this locally - in the path function it needs to be the actual name of the endpoint in Satellite - which is a GET on ansible_roles, not list_ansible_roles

Reply:
PRT run passed with these changes in my open PR. Good catch @sambible